lib/rules/lint-no-action-helper.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const Rule = require('./base');
+
+const ERROR_MESSAGE =
+  'Do not use the `action` helper. Instead, use the `on` modifier and `fn` helper.';
```

Comment:
I'd like for the error message to show how they were using `action` including the curlies or paren. We should also avoid calling it a helper (since sometimes it's a helper and sometimes it's a modifier).